tools/raml2html/README.md

```diff
@@ -4,15 +4,20 @@ Tool for generating static HTML from RAML definitions.
 
 ## Installation
 
-Install required dependencies before use. Go to raml2html root directory and run:
+PHP 7.2 and [Composer](https://getcomposer.org/) are required.
 
-```
-composer install
+To install required dependencies, go to raml2html directory and run:
+
+```sh
+composer install;
 ``` 
 
-To generate static HTML from RAML definitions, use the following code:
+## Usage
 
+To generate static HTML from RAML definitions, use the following code from project root:
 
 ```sh
-php tools/raml2html/raml2html.php build --non-standard-http-methods=COPY,MOVE,PUBLISH,SWAP -t default -o docs/api/rest_api_reference/output/ docs/api/rest_api_reference/input/ez.raml
+php tools/raml2html/raml2html.php build --non-standard-http-methods=COPY,MOVE,PUBLISH,SWAP -t default -o docs/api/rest_api/rest_api_reference/output/ docs/api/rest_api/rest_api_reference/input/ez.raml
```

Comment:
- `mv docs/api/rest_api/rest_api_reference/output/rest_api_reference.html docs/api/rest_api/rest_api_reference/rest_api_reference.html` could be added to this README.md
- docs/api/rest_api/rest_api_reference/output/ could be added to .gitignore
- To generate directly at the final place could be proposed (`php tools/raml2html/raml2html.php build --non-standard-http-methods=COPY,MOVE,PUBLISH,SWAP -t default -o docs/api/rest_api/rest_api_reference/ docs/api/rest_api/rest_api_reference/input/ez.raml`)